common/Generieke-Datatypen-Gemeenten.yaml

```diff
@@ -39,6 +39,38 @@ components:
       pattern: ^[0-9]{20}$
       description: |
                     Het Organisatie-identificatienummer (OIN) is een uniek nummer dat Logius kan toekennen aan organisaties om zich te kunnen identificeren, authentiseren en of autoriseren bij digitaal berichtenverkeer binnen en met de overheid.
+    Point:
+      $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/pointGeoJSON.yaml'
+    LineString:
+      $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/linestringGeoJSON.yaml'
+    Polygon:
+      $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/polygonGeoJSON.yaml'
+    MultiPoint:
+      $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/multipointGeoJSON.yaml'
+    MultiLineString:
+      $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/multilinestringGeoJSON.yaml'
+    MultiPolygon:
+      $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/multiPolygonGeoJSON.yaml'
+    GeoJSONGeometrie:
+      title: GeoJSON Geometry
```

Comment:
Wijzig de titel nog even in 'GeoJSON Geometrie'.

Reply:
Done